### PR TITLE
Update description about serviceaccount token controller

### PR DESCRIPTION
**Pull request by CaoShuFeng**
This change specifies how to enable token controller in
controller-manager.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/3166)

### CODE REVIEW

File: docs/admin/service-accounts-admin.md [hunk 1]
```diff
@@ -64,6 +64,11 @@ TokenController runs as part of controller-manager. It acts asynchronously. It:
 - observes secret addition, and ensures the referenced ServiceAccount exists, and adds a token to the secret if needed
 - observes secret deletion and removes a reference from the corresponding ServiceAccount if needed
 
+Note that token controller feature in controller-manager need a service account private key file passed by
```
Comment (chenopis) [inline docs/admin/service-accounts-admin.md]: Suggested edit:
You must pass a service account private key file to the token controller in the controller-manager by using the \`--service-account-private-key-file\` option. The private key will be used to sign generated service account tokens. Similarly, you must pass the corresponding public key to the kube-apiserver using the \`--service-account-key-file\` option. The public key will be used to verify the tokens during authentication.

Comment (CaoShuFeng) [inline docs/admin/service-accounts-admin.md]: Done.

**Comment by lavalamp** [comment]
We have GOT to make these flag names clearer in the future.

**Comment by chenopis** [comment]
@lavalamp Can you Tech LGTM this?

**Comment by lavalamp** [comment]
I already did?

**Comment by chenopis** [comment]
@lavalamp D'oh! Sorry, I didn't see the label change. :(

**Pull request merged**